src/components/VInputField/VInputField.vue

```diff
@@ -1,11 +1,12 @@
 <template>
   <div
-    class="input-field group flex flex-row items-center hover:bg-dark-charcoal-06 focus-within:bg-dark-charcoal-06 group-hover:bg-dark-charcoal-06 h-12 p-0.5px focus-within:p-0 border focus-within:border-1.5 border-dark-charcoal-20 rounded-sm overflow-hidden focus-within:border-pink"
+    class="input-field group flex flex-row items-center hover:bg-dark-charcoal-06 focus-within:bg-dark-charcoal-06 group-hover:bg-dark-charcoal-06 p-0.5px focus-within:p-0 border focus-within:border-1.5 border-dark-charcoal-20 rounded-sm overflow-hidden focus-within:border-pink"
     :class="[
       {
         // Padding is set to 1.5px to accommodate the border that will appear later.
         'border-s-0 ps-1.5px rounded-s-none': connectionSides.includes('start'),
         'border-e-0 pe-1.5px rounded-e-none': connectionSides.includes('end'),
+        [sizeClass]: true,
```

Comment:
I learned reviewing one of @dhruvkb's PRs the other day that you can mix the array and object class binding syntax. To avoid having to do `[sizeClass]: true` you can just put it into the array next to the object instead of inside the object.

Reply:
This is nice, Thank you, @sarayourfriend !